lib/usecases/log/GetAllLogsUseCase.js

```diff
@@ -54,18 +52,26 @@ class GetAllLogsUseCase {
             }
 
             if (tag) {
+                const subQueryBuilder = new QueryBuilder();
+                subQueryBuilder.where('tagId').oneOf(...tag.values);
+                subQueryBuilder.orderBy('logId', 'asc');
+                let logTags;
+
                 switch (tag.operation) {
                     case 'and':
-                        queryBuilder
-                            .setModel(Log)
-                            .whereAssociation('tags', 'id').allOf(...tag.values);
+                        logTags = await await TransactionHelper
```

Comment:
These should be include the in the other transaction helper.

Reply:
What do you mean exactly? Include what in which other transaction helper?

Reply:
Included in the transaction helper on line 89

Reply:
Ahh okay, gotcha